application/server/notification/scan_notifier.go

```diff
@@ -90,6 +90,26 @@ func (n *scanNotifier) sendSuccess(pr product.Product, folderPath string, issues
 				Lines:     lines,
 			})
 		}
+
+		markers := make([]lsp.Marker, 0, len(additionalData.Markers))
+		for _, marker := range additionalData.Markers {
+			positions := make([]lsp.MarkerPosition, 0)
```

Comment:
you could initialize it directly with size `len(marker.Pos)`

Reply:
I'd change it but @asaf92 has merged the PR already... 😅